data/sql/updates/pending_db_world/rev_1616884299081820400.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1616884299081820400');
+
+-- Updated gossip_menu for Doctor Gregory Victor <Trauma Surgeon> 
+UPDATE `creature_template` SET `gossip_menu_id`='5381' WHERE  `entry`=12920;
```

Comment:
the gossip menu id should not be a string
you should write `gossip_menu_id`=5381